mlrun/model_monitoring/stream_processing_fs.py

```diff
@@ -622,9 +619,7 @@ def do(self, full_event):
         self.resume_state(endpoint_id)
 
         # Handle errors coming from stream
-        found_errors = self.handle_errors(endpoint_id, event)
-        if found_errors:
-            return None
+        self.handle_errors(endpoint_id, event)
```

Comment:
in my opinion the function name / flow is confusing - we call function handle_errors() even when there are no errors..
suggest to move ```if "error" in event:``` outside of the function